model/Export/QTIPackedItemExporter.php

```diff
@@ -199,11 +199,14 @@ public function exportManifest($options = [], $exportResult = [])
             }
 
             $manifest = $this->getManifest();
-            $this->getMetadataExporter()->export($this->getItem(), $manifest);
-            $this->injectMetadataToManifest($manifest, $this->getItem());
-            $this->setManifest($manifest);
 
+            //APIP does not support metadata at this moment
+            if(!isset($options['apip']) && $options['apip'] !== true){
```

Comment:
Shouldn it be OR instead of AND ? If `apip` is not set OR `apip` has value `false` ?  Then do metadata stuff?

Reply:
This is incorrect. With an OR, the evaluation of the second member is done only if the first member is false, which means here that $options['apip'] is set. With AND, the second member will be evaluated only if the first member is true, which means that $options['apip'] is not set. Karol is right, it should be OR instead of AND.